wagtailcodeblock/blocks.py

```diff
@@ -1,7 +1,18 @@
+from django.conf import settings
 from django.forms import Media
-from wagtail.wagtailcore.blocks import StructBlock, TextBlock, ChoiceBlock
+from django.utils.translation import ugettext_lazy as _
 
-from .settings import get_language_choices
+from wagtail.wagtailcore.blocks import (
+    StructBlock,
+    TextBlock,
+    ChoiceBlock,
+    RichTextBlock

```

Comment:
I don't think we actually need this RichTextBlock, do we?

Reply:
I was thinking about it, maube it will be good for edit indentation and other thinks, but for now, we don't need it.

Reply:
Unfortunately, I don't think that'll work as the RichTextBlock (which I wish would just go away!) introduces HTML behind the scenes which would break the syntax highlighting.